test/utils_test.js

```diff
@@ -19,3 +19,27 @@ exports.testMergeDontPresereParameterWhenUndefinedOrNull =
     assert.equal(2, result2.retries);
   }
 
+exports.testMakeAmountInitialAndExpiration = function(beforeExit, assert) {
+    var extras, buf, fixture;
+    extras = utils.makeAmountInitialAndExpiration(1, 1, 1);
+    fixture = new Buffer('0000000000000001000000000000000100000001', 'hex');
+    assert.equal(20, extras.length);
+    assert.equal(fixture.toString('hex'), extras.toString('hex'));
+    buf = new Buffer(extras);
+    assert.equal(20, buf.length);
+    assert.equal(fixture.toString('hex'), buf.toString('hex'));
+
+    extras = utils.makeAmountInitialAndExpiration(255, 1, 1);
+    fixture = new Buffer('00000000000000ff000000000000000100000001', 'hex');
+    assert.equal(20, extras.length);
+    assert.equal(fixture.toString('hex'), extras.toString('hex'));
+    buf = new Buffer(extras);
+    assert.equal(20, buf.length);
+    assert.equal(fixture.toString('hex'), buf.toString('hex'));
+}
+
+exports.testMakeRequestBuffer = function(beforeExit, assert) {
```

Comment:
OK, this test is clearly useful for ensuring the result of `makeAmountInitialAndExpiration` feeds in properly to `makeRequestBuffer`, but:

  * it only tests the length of the `extras`
  * it's name implies it tests more functionality from the method

I suggest either changing the name of the test to be more specific, or maybe adding a TODO comment that it should test the rest of the output of `makeRequestBuffer`.